docs build fails (requirements.txt is missing tskit_book_theme, at least)

I ran `pip install -U -r python/requirements/development.txt`  then got the following while building the docs.
```
(venv) t490:tskit $ make -C docs
...
<snip long irrelevant doxygen stuff>
...
Running Sphinx v4.5.0
making output directory... done
[etoc] Changing master_doc to 'introduction'
[etoc] Excluded 4 extra file(s) not in toc

Theme error:
no theme named 'tskit_book_theme' found (missing theme.conf?)
Traceback (most recent call last):
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/jupyter_book/sphinx.py", line 114, in build_sphinx
    app = Sphinx(
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/sphinx/application.py", line 256, in __init__
    self._init_builder()
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/sphinx/application.py", line 314, in _init_builder
    self.builder.init()
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/sphinx/builders/html/__init__.py", line 212, in init
    self.init_templates()
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/sphinx/builders/html/__init__.py", line 260, in init_templates
    self.theme = theme_factory.create(themename)
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/sphinx/theming.py", line 241, in create
    raise ThemeError(__('no theme named %r found (missing theme.conf?)') % name)
sphinx.errors.ThemeError: no theme named 'tskit_book_theme' found (missing theme.conf?)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/grg/src/tskit/venv/bin/jupyter-book", line 8, in <module>
    sys.exit(main())
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/jupyter_book/cli/main.py", line 323, in build
    builder_specific_actions(
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/jupyter_book/cli/main.py", line 531, in builder_specific_actions
    raise RuntimeError(_message_box(msg, color="red", doprint=False)) from result
RuntimeError:
===============================================================================

There was an error in building your book. Look above for the cause.

===============================================================================

make: *** [Makefile:16: dev] Error 1
make: Leaving directory '/home/grg/src/tskit/docs'
```

So I pip-installed `tskit_book_theme`, but then got a new error. I probably messed something up somewhere?

```
...
<snip long irrelevant doxygen stuff>
...

Running Sphinx v4.5.0
making output directory... done
[etoc] Changing master_doc to 'introduction'
[etoc] Excluded 4 extra file(s) not in toc
myst v0.15.2: MdParserConfig(renderer='sphinx', commonmark_only=False, enable_extensions=['colon_fence', 'deflist', 'dollarmath', 'substitution'], dmath_allow_labels=True, dmath_allow_space=True, dmath_allow_digits=True, dmath_double_inline=False, update_mathjax=True, mathjax_classes='tex2jax_process|mathjax_process|math|output_area', disable_syntax=[], url_schemes=['mailto', 'http', 'https'], heading_anchors=None, heading_slug_func=None, html_meta=[], footnote_transition=True, substitutions=[], sub_delimiters=['{', '}'], words_per_minute=200)
loading intersphinx inventory from https://docs.python.org/3/objects.inv...
loading intersphinx inventory from https://tskit.dev/tutorials/objects.inv...
loading intersphinx inventory from https://stdpopsim.readthedocs.io/en/stable/objects.inv...
loading intersphinx inventory from https://tskit.dev/pyslim/docs/latest/objects.inv...
loading intersphinx inventory from https://tskit.dev/msprime/docs/stable/objects.inv...
loading intersphinx inventory from https://numpy.org/doc/stable/objects.inv...
intersphinx inventory has moved: https://stdpopsim.readthedocs.io/en/stable/objects.inv -> https://popsim-consortium.github.io/stdpopsim-docs/stable/objects.inv
[autosummary] generating autosummary for: c-api.rst, changelogs.rst, cli.md, data-model.md, development.md, export.md, file-formats.md, glossary.md, ibd.md, installation.md, introduction.md, metadata.md, provenance.md, python-api.md, quickstart.md, stats.md, topological-analysis.md
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 17 source files that are out of date
updating environment: executing outdated notebooks... Executing: /home/grg/src/tskit/docs/ibd.md
Executing: /home/grg/src/tskit/docs/development.md
Execution Succeeded: /home/grg/src/tskit/docs/development.md
Executing: /home/grg/src/tskit/docs/file-formats.md
Executing: /home/grg/src/tskit/docs/introduction.md
Execution Succeeded: /home/grg/src/tskit/docs/introduction.md
Executing: /home/grg/src/tskit/docs/metadata.md
Execution Succeeded: /home/grg/src/tskit/docs/metadata.md
Executing: /home/grg/src/tskit/docs/cli.md
Execution Succeeded: /home/grg/src/tskit/docs/cli.md
Executing: /home/grg/src/tskit/docs/export.md
Executing: /home/grg/src/tskit/docs/topological-analysis.md
Executing: /home/grg/src/tskit/docs/data-model.md
Executing: /home/grg/src/tskit/docs/quickstart.md
Executing: /home/grg/src/tskit/docs/glossary.md
Execution Succeeded: /home/grg/src/tskit/docs/glossary.md
Executing: /home/grg/src/tskit/docs/provenance.md
Execution Succeeded: /home/grg/src/tskit/docs/provenance.md
Executing: /home/grg/src/tskit/docs/stats.md
Executing: /home/grg/src/tskit/docs/installation.md
Execution Succeeded: /home/grg/src/tskit/docs/installation.md
Executing: /home/grg/src/tskit/docs/python-api.md
Execution Succeeded: /home/grg/src/tskit/docs/python-api.md
done
[new config] 17 added, 0 changed, 0 removed
reading sources... [100%] topological-analysis
ERROR: Execution Failed: /home/grg/src/tskit/docs/ibd.md
ERROR: Execution Failed: /home/grg/src/tskit/docs/file-formats.md
ERROR: Execution Failed: /home/grg/src/tskit/docs/export.md
ERROR: Execution Failed: /home/grg/src/tskit/docs/topological-analysis.md
ERROR: Execution Failed: /home/grg/src/tskit/docs/data-model.md
ERROR: Execution Failed: /home/grg/src/tskit/docs/quickstart.md
ERROR: Execution Failed: /home/grg/src/tskit/docs/stats.md
ERROR: Couldn't find cache key for notebook file data-model.md. Outputs will not be inserted.
  Last execution failed with traceback saved in /home/grg/src/tskit/docs/_build/html/reports/data-model.log
ERROR: Couldn't find cache key for notebook file export.md. Outputs will not be inserted.
  Last execution failed with traceback saved in /home/grg/src/tskit/docs/_build/html/reports/export.log
ERROR: Couldn't find cache key for notebook file file-formats.md. Outputs will not be inserted.
  Last execution failed with traceback saved in /home/grg/src/tskit/docs/_build/html/reports/file-formats.log
ERROR: Couldn't find cache key for notebook file ibd.md. Outputs will not be inserted.
  Last execution failed with traceback saved in /home/grg/src/tskit/docs/_build/html/reports/ibd.log
ERROR: Couldn't find cache key for notebook file quickstart.md. Outputs will not be inserted.
  Last execution failed with traceback saved in /home/grg/src/tskit/docs/_build/html/reports/quickstart.log
ERROR: Couldn't find cache key for notebook file stats.md. Outputs will not be inserted.
  Last execution failed with traceback saved in /home/grg/src/tskit/docs/_build/html/reports/stats.log
ERROR: Couldn't find cache key for notebook file topological-analysis.md. Outputs will not be inserted.
  Last execution failed with traceback saved in /home/grg/src/tskit/docs/_build/html/reports/topological-analysis.log
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [100%] topological-analysis
generating indices... genindex py-modindex done
highlighting module code... [100%] tskit.util
writing additional pages... search done
copying images... [100%] _static/four_leaf_tree_shapes.png
copying static files... done
copying extra files... done
dumping search index in English (code: en)... done
dumping object inventory... done
build finished with problems, 14 warnings.
[etoc] missing index.html written as redirect to 'introduction.html'

===============================================================================

Building your book, returns a non-zero exit code (1). Look above for the cause.

===============================================================================

Error occured; showing saved reports
Traceback (most recent call last):
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/client.py", line 1093, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/usr/lib/python3.10/asyncio/base_events.py", line 646, in run_until_complete
    return future.result()
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/client.py", line 559, in async_execute
    await self.async_execute_cell(
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/client.py", line 854, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/client.py", line 756, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import io

import tskit
from IPython.display import SVG

nodes = """\
id      is_sample   time
0       1           0
1       1           0
2       1           0
3       1           0
4       1           0
5       0           1
6       0           2
7       0           3
"""
edges = """\
left    right   parent  child
0       60      5       4,3
0       40      6       2
0       60      6       1,0
20      40      6       5
0       20      7       5
40      60      7       5
0       60      7       6
40      60      7       2
"""
ts = tskit.load_text(
    nodes=io.StringIO(nodes), edges=io.StringIO(edges), strict=False
)

SVG(ts.first().draw_svg(time_scale="rank"))
------------------

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
/tmp/ipykernel_194583/3371847887.py in <module>
     26 40      60      7       2
     27 """
---> 28 ts = tskit.load_text(
     29     nodes=io.StringIO(nodes), edges=io.StringIO(edges), strict=False
     30 )

~/src/tskit/python/tskit/trees.py in load_text(nodes, edges, sites, mutations, individuals, populations, migrations, sequence_length, strict, encoding, base64_metadata)
   3772         )
   3773     tc.sort()
-> 3774     return tc.tree_sequence()
   3775
   3776

~/src/tskit/python/tskit/tables.py in tree_sequence(self)
   3339         if not self.has_index():
   3340             self.build_index()
-> 3341         return tskit.TreeSequence.load_tables(self)
   3342
   3343     def simplify(

~/src/tskit/python/tskit/trees.py in load_tables(cls, tables, build_indexes)
   4010         ts = _tskit.TreeSequence()
   4011         ts.load_tables(tables._ll_tables, build_indexes=build_indexes)
-> 4012         return TreeSequence(ts)
   4013
   4014     def dump(self, file_or_path, zlib_compression=False):

~/src/tskit/python/tskit/trees.py in __init__(self, ll_tree_sequence)
   3908         # tables here (and remove the low-level boilerplate).
   3909         llts = self._ll_tree_sequence
-> 3910         self._individuals_flags = llts.individuals_flags
   3911         self._nodes_time = llts.nodes_time
   3912         self._nodes_flags = llts.nodes_flags

AttributeError: '_tskit.TreeSequence' object has no attribute 'individuals_flags'
AttributeError: '_tskit.TreeSequence' object has no attribute 'individuals_flags'

Traceback (most recent call last):
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/client.py", line 1093, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/usr/lib/python3.10/asyncio/base_events.py", line 646, in run_until_complete
    return future.result()
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/client.py", line 559, in async_execute
    await self.async_execute_cell(
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/client.py", line 854, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/client.py", line 756, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import sys
import msprime
ts = msprime.sim_ancestry(
    samples=3, ploidy=2, sequence_length=10, random_seed=2)
ts = msprime.sim_mutations(ts, rate=0.1, random_seed=2)
ts.write_vcf(sys.stdout)
------------------

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
/tmp/ipykernel_194512/126153089.py in <module>
      1 import sys
      2 import msprime
----> 3 ts = msprime.sim_ancestry(
      4     samples=3, ploidy=2, sequence_length=10, random_seed=2)
      5 ts = msprime.sim_mutations(ts, rate=0.1, random_seed=2)

~/src/tskit/venv/lib/python3.10/site-packages/msprime/ancestry.py in sim_ancestry(samples, demography, sequence_length, discrete_genome, recombination_rate, gene_conversion_rate, gene_conversion_tract_length, population_size, ploidy, model, initial_state, start_time, end_time, record_migrations, record_full_arg, num_labels, random_seed, num_replicates, replicate_index, record_provenance)
   1224         random_seed=random_seed,
   1225     )
-> 1226     return _wrap_replicates(
   1227         sim,
   1228         num_replicates=num_replicates,

~/src/tskit/venv/lib/python3.10/site-packages/msprime/ancestry.py in _wrap_replicates(simulator, num_replicates, replicate_index, provenance_dict, mutation_rate)
    658     )
    659     if replicate_index is not None:
--> 660         deque = collections.deque(iterator, maxlen=1)
    661         return deque.pop()
    662     else:

~/src/tskit/venv/lib/python3.10/site-packages/msprime/ancestry.py in run_replicates(self, num_replicates, mutation_rate, provenance_dict)
   1516                 )
   1517                 tables.provenances.add_row(replicate_provenance)
-> 1518             ts = tables.tree_sequence()
   1519             yield ts
   1520             self.reset()

~/src/tskit/python/tskit/tables.py in tree_sequence(self)
   3339         if not self.has_index():
   3340             self.build_index()
-> 3341         return tskit.TreeSequence.load_tables(self)
   3342
   3343     def simplify(

~/src/tskit/python/tskit/trees.py in load_tables(cls, tables, build_indexes)
   4010         ts = _tskit.TreeSequence()
   4011         ts.load_tables(tables._ll_tables, build_indexes=build_indexes)
-> 4012         return TreeSequence(ts)
   4013
   4014     def dump(self, file_or_path, zlib_compression=False):

~/src/tskit/python/tskit/trees.py in __init__(self, ll_tree_sequence)
   3908         # tables here (and remove the low-level boilerplate).
   3909         llts = self._ll_tree_sequence
-> 3910         self._individuals_flags = llts.individuals_flags
   3911         self._nodes_time = llts.nodes_time
   3912         self._nodes_flags = llts.nodes_flags

AttributeError: '_tskit.TreeSequence' object has no attribute 'individuals_flags'
AttributeError: '_tskit.TreeSequence' object has no attribute 'individuals_flags'

Traceback (most recent call last):
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/client.py", line 1093, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/usr/lib/python3.10/asyncio/base_events.py", line 646, in run_until_complete
    return future.result()
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/client.py", line 559, in async_execute
    await self.async_execute_cell(
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/client.py", line 854, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/client.py", line 756, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# TODO once https://github.com/tskit-dev/tskit/issues/1824 is solved
# change the individual table to include some with blank parents / locations
import io

import tskit
from IPython.display import SVG

individuals = """\
flags       location     parents
0           0.5,1.2      -1,-1
0           1.0,3.4      0,-1
0           3.5,6.3      0,1
0           0.5          -1,-1
0           0.5,0.5      2,3
"""

nodes = """\
is_sample   individual   time
1           0            0.0
1           0            0.0
0           -1           2.0
0           -1           3.0
"""
edges = """\
left   right   parent  child
0.0    7.0     2       0
0.0    7.0     2       1
7.0    10.0    3       0
7.0    10.0    3       1
"""

sites = """\
position      ancestral_state
2.0           AT
4.0           A
"""

mutations = """\
site   node    derived_state    time    parent
0      0       A                0.5     -1
1      0       T                1.5     -1
1      1       A                1.0     1
"""

migrations = """\
left   right   node   source   dest   time
0.0    0.7     5      2        3      1.0
0.8    0.9     8      3        4      3.0
"""

populations = """\
id   metadata
0    cG9wMQ==
1    cG9wMg==
"""

ts = tskit.load_text(
    individuals=io.StringIO(individuals),
    nodes=io.StringIO(nodes),
    edges=io.StringIO(edges),
    sites=io.StringIO(sites),
    mutations=io.StringIO(mutations),
    # migrations=io.StringIO(migrations),  # uncomment when https://github.com/tskit-dev/tskit/issues/19 fixed
    populations=io.StringIO(populations),
    strict=False
)
SVG(ts.draw_svg(y_axis=True))
------------------

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
/tmp/ipykernel_194387/1826002553.py in <module>
     55 """
     56
---> 57 ts = tskit.load_text(
     58     individuals=io.StringIO(individuals),
     59     nodes=io.StringIO(nodes),

~/src/tskit/python/tskit/trees.py in load_text(nodes, edges, sites, mutations, individuals, populations, migrations, sequence_length, strict, encoding, base64_metadata)
   3772         )
   3773     tc.sort()
-> 3774     return tc.tree_sequence()
   3775
   3776

~/src/tskit/python/tskit/tables.py in tree_sequence(self)
   3339         if not self.has_index():
   3340             self.build_index()
-> 3341         return tskit.TreeSequence.load_tables(self)
   3342
   3343     def simplify(

~/src/tskit/python/tskit/trees.py in load_tables(cls, tables, build_indexes)
   4010         ts = _tskit.TreeSequence()
   4011         ts.load_tables(tables._ll_tables, build_indexes=build_indexes)
-> 4012         return TreeSequence(ts)
   4013
   4014     def dump(self, file_or_path, zlib_compression=False):

~/src/tskit/python/tskit/trees.py in __init__(self, ll_tree_sequence)
   3908         # tables here (and remove the low-level boilerplate).
   3909         llts = self._ll_tree_sequence
-> 3910         self._individuals_flags = llts.individuals_flags
   3911         self._nodes_time = llts.nodes_time
   3912         self._nodes_flags = llts.nodes_flags

AttributeError: '_tskit.TreeSequence' object has no attribute 'individuals_flags'
AttributeError: '_tskit.TreeSequence' object has no attribute 'individuals_flags'

Traceback (most recent call last):
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/client.py", line 1093, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/usr/lib/python3.10/asyncio/base_events.py", line 646, in run_until_complete
    return future.result()
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/client.py", line 559, in async_execute
    await self.async_execute_cell(
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/client.py", line 854, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/client.py", line 756, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import tskit
import io
from IPython.display import SVG

nodes = io.StringIO(
    """\
    id      is_sample   time
    0       1           0
    1       1           0
    2       1           0
    3       0           1
    4       0           2
    5       0           3
    """
)
edges = io.StringIO(
    """\
    left    right   parent  child
    2     10     3       0
    2     10     3       2
    0     10     4       1
    0     2      4       2
    2     10     4       3
    0     2      5       0
    0     2      5       4
    """
)
ts = tskit.load_text(nodes=nodes, edges=edges, strict=False)

SVG(ts.draw_svg())
------------------

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
/tmp/ipykernel_194330/1334198093.py in <module>
     26     """
     27 )
---> 28 ts = tskit.load_text(nodes=nodes, edges=edges, strict=False)
     29
     30 SVG(ts.draw_svg())

~/src/tskit/python/tskit/trees.py in load_text(nodes, edges, sites, mutations, individuals, populations, migrations, sequence_length, strict, encoding, base64_metadata)
   3772         )
   3773     tc.sort()
-> 3774     return tc.tree_sequence()
   3775
   3776

~/src/tskit/python/tskit/tables.py in tree_sequence(self)
   3339         if not self.has_index():
   3340             self.build_index()
-> 3341         return tskit.TreeSequence.load_tables(self)
   3342
   3343     def simplify(

~/src/tskit/python/tskit/trees.py in load_tables(cls, tables, build_indexes)
   4010         ts = _tskit.TreeSequence()
   4011         ts.load_tables(tables._ll_tables, build_indexes=build_indexes)
-> 4012         return TreeSequence(ts)
   4013
   4014     def dump(self, file_or_path, zlib_compression=False):

~/src/tskit/python/tskit/trees.py in __init__(self, ll_tree_sequence)
   3908         # tables here (and remove the low-level boilerplate).
   3909         llts = self._ll_tree_sequence
-> 3910         self._individuals_flags = llts.individuals_flags
   3911         self._nodes_time = llts.nodes_time
   3912         self._nodes_flags = llts.nodes_flags

AttributeError: '_tskit.TreeSequence' object has no attribute 'individuals_flags'
AttributeError: '_tskit.TreeSequence' object has no attribute 'individuals_flags'

Traceback (most recent call last):
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/client.py", line 1093, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/usr/lib/python3.10/asyncio/base_events.py", line 646, in run_until_complete
    return future.result()
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/client.py", line 559, in async_execute
    await self.async_execute_cell(
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/client.py", line 854, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/client.py", line 756, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import tskit

ts = tskit.load("data/basic_tree_seq.trees")  # Or generate using e.g. msprime.sim_ancestry()
ts  # In a Jupyter notebook this displays a summary table. Otherwise use print(ts)
------------------

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
/tmp/ipykernel_194615/335230046.py in <module>
      1 import tskit
      2
----> 3 ts = tskit.load("data/basic_tree_seq.trees")  # Or generate using e.g. msprime.sim_ancestry()
      4 ts  # In a Jupyter notebook this displays a summary table. Otherwise use print(ts)

~/src/tskit/python/tskit/trees.py in load(file, skip_tables, skip_reference_sequence)
   3198     :rtype: :class:`tskit.TreeSequence`
   3199     """
-> 3200     return TreeSequence.load(
   3201         file, skip_tables=skip_tables, skip_reference_sequence=skip_reference_sequence
   3202     )

~/src/tskit/python/tskit/trees.py in load(cls, file_or_path, skip_tables, skip_reference_sequence)
   4001                 skip_reference_sequence=skip_reference_sequence,
   4002             )
-> 4003             return TreeSequence(ts)
   4004         finally:
   4005             if local_file:

~/src/tskit/python/tskit/trees.py in __init__(self, ll_tree_sequence)
   3908         # tables here (and remove the low-level boilerplate).
   3909         llts = self._ll_tree_sequence
-> 3910         self._individuals_flags = llts.individuals_flags
   3911         self._nodes_time = llts.nodes_time
   3912         self._nodes_flags = llts.nodes_flags

AttributeError: '_tskit.TreeSequence' object has no attribute 'individuals_flags'
AttributeError: '_tskit.TreeSequence' object has no attribute 'individuals_flags'

Traceback (most recent call last):
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/client.py", line 1093, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/usr/lib/python3.10/asyncio/base_events.py", line 646, in run_until_complete
    return future.result()
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/client.py", line 559, in async_execute
    await self.async_execute_cell(
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/client.py", line 854, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/client.py", line 756, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
from IPython.display import Markdown
import msprime
import numpy as np

demography = msprime.Demography()
demography.add_population(name="A", initial_size=10_000)
demography.add_population(name="B", initial_size=10_000)
demography.set_symmetric_migration_rate(["A", "B"], 0.001)
ts = msprime.sim_ancestry(
    samples={"A": 2, "B": 2},
    sequence_length=1000,
    demography=demography,
    recombination_rate=2e-8,
    random_seed=12)
ts = msprime.sim_mutations(ts, rate=2e-8, random_seed=12)
Markdown(
    f"These examples use a tree sequence of {ts.num_samples} samples "
    f"in {ts.num_populations} populations, "
    f"with a sequence length of {int(ts.sequence_length)}. "
    f"There are {ts.num_trees} trees and "
    f"{ts.num_sites} variable sites in the tree sequence."
)
------------------

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
/tmp/ipykernel_194704/2967012165.py in <module>
      7 demography.add_population(name="B", initial_size=10_000)
      8 demography.set_symmetric_migration_rate(["A", "B"], 0.001)
----> 9 ts = msprime.sim_ancestry(
     10     samples={"A": 2, "B": 2},
     11     sequence_length=1000,

~/src/tskit/venv/lib/python3.10/site-packages/msprime/ancestry.py in sim_ancestry(samples, demography, sequence_length, discrete_genome, recombination_rate, gene_conversion_rate, gene_conversion_tract_length, population_size, ploidy, model, initial_state, start_time, end_time, record_migrations, record_full_arg, num_labels, random_seed, num_replicates, replicate_index, record_provenance)
   1224         random_seed=random_seed,
   1225     )
-> 1226     return _wrap_replicates(
   1227         sim,
   1228         num_replicates=num_replicates,

~/src/tskit/venv/lib/python3.10/site-packages/msprime/ancestry.py in _wrap_replicates(simulator, num_replicates, replicate_index, provenance_dict, mutation_rate)
    658     )
    659     if replicate_index is not None:
--> 660         deque = collections.deque(iterator, maxlen=1)
    661         return deque.pop()
    662     else:

~/src/tskit/venv/lib/python3.10/site-packages/msprime/ancestry.py in run_replicates(self, num_replicates, mutation_rate, provenance_dict)
   1516                 )
   1517                 tables.provenances.add_row(replicate_provenance)
-> 1518             ts = tables.tree_sequence()
   1519             yield ts
   1520             self.reset()

~/src/tskit/python/tskit/tables.py in tree_sequence(self)
   3339         if not self.has_index():
   3340             self.build_index()
-> 3341         return tskit.TreeSequence.load_tables(self)
   3342
   3343     def simplify(

~/src/tskit/python/tskit/trees.py in load_tables(cls, tables, build_indexes)
   4010         ts = _tskit.TreeSequence()
   4011         ts.load_tables(tables._ll_tables, build_indexes=build_indexes)
-> 4012         return TreeSequence(ts)
   4013
   4014     def dump(self, file_or_path, zlib_compression=False):

~/src/tskit/python/tskit/trees.py in __init__(self, ll_tree_sequence)
   3908         # tables here (and remove the low-level boilerplate).
   3909         llts = self._ll_tree_sequence
-> 3910         self._individuals_flags = llts.individuals_flags
   3911         self._nodes_time = llts.nodes_time
   3912         self._nodes_flags = llts.nodes_flags

AttributeError: '_tskit.TreeSequence' object has no attribute 'individuals_flags'
AttributeError: '_tskit.TreeSequence' object has no attribute 'individuals_flags'

Traceback (most recent call last):
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/client.py", line 1093, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/usr/lib/python3.10/asyncio/base_events.py", line 646, in run_until_complete
    return future.result()
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/client.py", line 559, in async_execute
    await self.async_execute_cell(
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/client.py", line 854, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/home/grg/src/tskit/venv/lib/python3.10/site-packages/nbclient/client.py", line 756, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import tskit
from IPython.display import SVG, display

tree = tskit.Tree.generate_balanced(10, arity=3)
display(SVG(tree.draw_svg()))

for order in ["preorder", "inorder", "postorder"]:
    print(f"{order}:\t", list(tree.nodes(order=order)))
------------------

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
/tmp/ipykernel_194551/377664874.py in <module>
      2 from IPython.display import SVG, display
      3
----> 4 tree = tskit.Tree.generate_balanced(10, arity=3)
      5 display(SVG(tree.draw_svg()))
      6

~/src/tskit/python/tskit/trees.py in generate_balanced(num_leaves, arity, span, branch_length, record_provenance, **kwargs)
   3071         :rtype: Tree
   3072         """
-> 3073         return combinatorics.generate_balanced(
   3074             num_leaves,
   3075             arity=arity,

~/src/tskit/python/tskit/combinatorics.py in generate_balanced(num_leaves, arity, span, branch_length, record_provenance, **kwargs)
    282         )
    283
--> 284     return tables.tree_sequence().first(**kwargs)
    285
    286

~/src/tskit/python/tskit/tables.py in tree_sequence(self)
   3339         if not self.has_index():
   3340             self.build_index()
-> 3341         return tskit.TreeSequence.load_tables(self)
   3342
   3343     def simplify(

~/src/tskit/python/tskit/trees.py in load_tables(cls, tables, build_indexes)
   4010         ts = _tskit.TreeSequence()
   4011         ts.load_tables(tables._ll_tables, build_indexes=build_indexes)
-> 4012         return TreeSequence(ts)
   4013
   4014     def dump(self, file_or_path, zlib_compression=False):

~/src/tskit/python/tskit/trees.py in __init__(self, ll_tree_sequence)
   3908         # tables here (and remove the low-level boilerplate).
   3909         llts = self._ll_tree_sequence
-> 3910         self._individuals_flags = llts.individuals_flags
   3911         self._nodes_time = llts.nodes_time
   3912         self._nodes_flags = llts.nodes_flags

AttributeError: '_tskit.TreeSequence' object has no attribute 'individuals_flags'
AttributeError: '_tskit.TreeSequence' object has no attribute 'individuals_flags'

make: *** [Makefile:16: dev] Error 1
make: Leaving directory '/home/grg/src/tskit/docs'

```

--- a/python/requirements/development.txt
+++ b/python/requirements/development.txt
@@ -35,6 +35,7 @@ sphinx-issues
 sphinx-jupyterbook-latex
 sphinxcontrib-prettyspecialmethods
 tqdm
+tskit-book-theme
 pydata_sphinx_theme>=0.7.2
 svgwrite>=1.1.10
 xmlunittest

